src/internal/events/events.go

```diff
@@ -38,6 +38,7 @@ const (
 	DataStored       = "data_stored"
 	Duration         = "duration"
 	EndTime          = "end_time"
+	HumanDuration    = "human_duration"
```

Comment:
What is the rationale of having two of these fields? Not sure I understand the difference between the two.

Reply:
From the slack thread you weren't able to make much immediate use of the duration value, I added the HumanDuration to give you something easily read.  However, it's not going to be easy to query against, and I assume in the future you'll want to query for data points like "count of events that took longer than X", which is why I kept the old value, too.